--- kmymoney-4.6.1/libkdchart/src/KDChartBackgroundAttributes.cpp.bero	2012-01-16 20:53:52.809328947 +0100
+++ kmymoney-4.6.1/libkdchart/src/KDChartBackgroundAttributes.cpp	2012-01-16 20:54:29.486326353 +0100
@@ -150,7 +150,7 @@ QDebug operator<<(QDebug dbg, const KDCh
 	<< "visible="<<ba.isVisible()
 	<< "brush="<<ba.brush()
 	<< "pixmapmode="<<ba.pixmapMode()
-	<< "pixmap="<<ba.pixmap()
+/*	<< "pixmap="<<ba.pixmap() <- Ambiguous overload */
 	<< ")";
     return dbg;
 }
