diff -uraN kmymoney-5.1.1/CMakeLists.txt omv-kmymoney-5.1.1/CMakeLists.txt
--- kmymoney-5.1.1/CMakeLists.txt	2020-12-20 10:29:57.000000000 +0100
+++ omv-kmymoney-5.1.1/CMakeLists.txt	2021-01-02 14:53:00.554737831 +0100
@@ -84,7 +84,7 @@
 endif()
 
 find_package(Qt5 5.6 REQUIRED COMPONENTS
-  Core DBus Widgets Svg Xml Test PrintSupport)
+  Core DBus Widgets Svg Xml Test PrintSupport WebKitWidgets)
 find_package(Qt5 5.6 OPTIONAL_COMPONENTS
   Sql Concurrent QuickWidgets)
 
